### DISCUSSION TITLE
Discord or IRC channel instead of Github Discussions?

### DISCUSSION THREAD

**Discussion by savolla** [tesseract-robotics/tesseract#767] [Ideas]
Is there any discussion channel other than github discussions? It would be great if you guys create a Discord/IRC/Matrix channels with necessary sub channels like *#help", "#general-discussion" etc. Moveit has it's own Discord channel and it's very active. People might have small questions or issues that are not worth for opening a new issue or discussion. But asking them on these platforms is quick and easy. Also this would help people to watch development process and project roadmap from closer perspective. Thank you!

**Comment by Levi-Armstrong** [comment]
Currently yes, but mostly because the others you have mentioned have not been requested or discussed. I think having all three may be to much to monitor but would Discord be the best or recommended option to start with?

**Reply by savolla** [reply]
Discord would be great. But since you said there is already a place, Discord might not be necessary anymore. Where do you guys communicate or discuss things at the moment? Can I have the link or invite?

**Reply by Levi-Armstrong** [reply]
I set up Discord under Tesseract Robotics. Let me know if you can find it and gain access. I have never used this before but looks very similar to slack. Also let me know if you have any recommendation on changes.

**Reply by savolla** [reply]
Sure! I'm also not an expert when it comes to Discord :) I want to make my point clear that this does not need to be exactly Discord. If you guys are more comfortable with Slack then let it be Slack. By the way I'm not able to find "Tesseract Robotics" on Discord's public channels.

**Reply by Levi-Armstrong** [reply]
I looked into slack but it was more difficult I thought at making it publicly accessible than Discord.

**Reply by Levi-Armstrong** [reply]
Try this link https://discord.gg/E9zttd5uun

**Comment by marip8** [comment]
Another option would be to use stack overflow. The `tesseract` tag is already being used by Tesseract OCR, but we could specify a new tag name like `tesseract-robotics` to collate discussions/questions. One benefit of stack overflow would be that discussions probably show up better in web searches than they would in a discord or slack channel.

**Reply by Levi-Armstrong** [reply]
I think they are more interested in live chatting capability. I believe stack overflow is very similar to the GitHub discussion but probably not as publicly visible.

**Comment by savolla** [comment]
We have a [Discord Channel](https://discord.gg/E9zttd5uun) now. Thanks @Levi-Armstrong !

**Comment by Levi-Armstrong** [comment]
Here is a [invite link](https://discord.gg/twggWySTQE) that does not expire.
